docs/self-managed/zeebe-deployment/exporters/camunda-exporter.md

```diff
@@ -0,0 +1,189 @@
+| variableSizeThreshold | Defines a threshold for variable size. Variables exceeding this threshold are split into two properties: `FULL_VALUE` (full content, not indexed) and `VALUE` (truncated content, indexed). | 8191    |
+| shardsByIndexName     | A map where the key is the index name and the value is the number of shards, allowing you to override the default `numberOfShards` setting for specific indices.                            |         |
+| replicasByIndexName   | A map where the key is the index name and the value is the number of replicas, allowing you to override the default `numberOfReplicas` setting for specific indices.                        |         |
+
+</TabItem>
+
+<TabItem value="bulk">
+
+To avoid too many expensive requests to the Elasticsearch/Opensearch cluster, the exporter performs batch
+updates by default. The size of the batch, along with how often it should be flushed (regardless of
+size) can be controlled by configuration.
+
+| Option | Description                                                                                                                                                    | Default |
+| ------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| delay  | Delay, in seconds, before force flush of the current batch. This ensures that even when we have low traffic of records, we still export every once in a while. | `5`     |
+| size   | The amount of records a batch should have before we flush the batch                                                                                            | `1000`  |
+
+With the default configuration, the exporter will aggregate records and flush them to Elasticsearch/Opensearch:
+
+1. When it has aggregated 1000 records. 2. Five seconds have elapsed since the last flush (regardless of how many records were aggregated).
+
+</TabItem>
+
+<TabItem value="retention">
+
+A retention policy can be set up to delete old data.
+When enabled, this creates an Index Lifecycle Management (ILM) Policy that deletes the data after the specified
+`minimumAge`.
+All index templates created by this exporter apply the created ILM Policy.
+
+| Option     | Description                                                                  | Default |
+| ---------- | ---------------------------------------------------------------------------- | ------- |
+| enabled    | If `true` the ILM Policy is created and applied to the index templates       | `false` |
+| minimumAge | Specifies how old the data must be, before the data is deleted as a duration | `30d`   |
+| policyName | The name of the created and applied ILM policy                               |         |
```

Comment:
❓ There is a default value for the policy name right?

Reply:
we used to have a default in zeebe, but here not.

Reply:
Isn;t this an issue then? If I enable this and have no name it would fail no? I need a default name I think

Reply:
Not really an issue, if you enable retention, you should give a name to the policy. it's not consistent with other exporters, but not an issue.

I can't remember if that was a decision, but I guess it was removed when there was a confusion whether we will have different policies or not.

Anyway, we can bring back a default value, but here I'm just documenting what there is right now.